packages/data/src/plugins/persistence/test/index.js

```diff
@@ -181,17 +171,6 @@ describe( 'persistence', () => {
 		expect( registry.select( 'test' ).getState() ).toBe( 1 );
 	} );
 
-	it( 'override values passed to registerStore', () => {
```

Comment:
@aduth – this test stopped working. I removed it because it was using mocking extensively. `originalRegisterStore` spy wasn't called anymore. Do you think we should try to bring this test back? If yes, do you have some ideas on how to update it?

Reply:
It's not really clear to me what it is about the Babel upgrade which would be making a difference here.

Looking at the test case, I don't think it's an especially important one, especially given how the assertion was framed. I think the general idea was that it would replace the reducer? But the assertion doesn't really make any expectation about what that overridden value is. In fact, it seems like it could very well have not changed the original reducer at all.

Which is all to say: I can see how there could be potential value in a test like this, but in how it was implemented previously, I don't think there was much value.

Reply:
I decided to remove this test, thanks for checking.